extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -160,7 +177,9 @@ export class AmpSidebar extends AMP.BaseElement {
         }, ANIMATION_TIMEOUT);
       });
     });
-    this.getHistory_().push(this.close_.bind(this)).then(historyId => {
+    this.getHistory_().push(() => {
+      this.close_();
```

Comment:
More of the "pop is done" - I don't think the same problem happens with pushing since there's no `pushstate` event.